prism/src/main/webapp/WEB-INF/web.xml

```diff
@@ -41,6 +41,15 @@
     </filter>
 
     <filter>
+        <filter-name>csrf</filter-name>
+        <filter-class>org.apache.falcon.security.FalconCSRFFilter</filter-class>
+        <init-param>
+            <param-name>methods-to-ignore</param-name>
+            <param-value>GET</param-value>
```

Comment:
Good idea. Yes, we can customize the header (e.g. add "falcon-" to header). Header, methods and user agents are the options provided in RestCsrfPreventionFilter currently.